### PR TITLE
Added test for doGetFromDatastore

### PR DESCRIPTION
**Pull request by smarquez-jpg**

### CODE REVIEW

File: portfolio/src/test/java/com/google/sps/DatastoreUtilsTest.java [hunk 1]
```diff
@@ -188,4 +188,11 @@ public void PutBlogsInDatastoreTest_withoutPost() {
     assertEquals(0, ds.prepare(new Query(BlogConstants.BLOG_ENTITY_KIND)).countEntities(withLimit(5)));
   }
 
+  @Test
+  public void testDoGetFromDatastore() {
+    createDatastoreEntities();
+    LinkedList<BlogMessage> actual = DatastoreUtils.doGetFromDatastore(1);
+
+    Assert.assertEquals(1, actual.size());
```
Comment (martiKallas) [inline portfolio/src/test/java/com/google/sps/DatastoreUtilsTest.java]: You need some extra checks here. You want this to fail if doGetFromDatastore changes in an unwanted way. doGetFromDatastore has 3 functions that it calls: putRepliesWithPosts, updateTagsToSearch, and sortAndLoadfromBlogHashmap. You will need to check the output from doGetFromDatastore to confirm that each of these are called.

**Review by martiKallas — CHANGES_REQUESTED** [review]
Thanks for starting this, Steven! Feel free to get help from your podmates too.

**Review by Andrewmob2703-zz — COMMENTED** [review]
I probably should have tested this since I added new functionality.
New Tests Added.

**Review by martiKallas — APPROVED** [review]
Nice changes!

I still don't think this tests if replies get attached to posts, but feel free to submit in the interest of time.

**Comment by Andrewmob2703-zz** [comment]
> Nice changes!
> 
> I still don't think this tests if replies get attached to posts, but feel free to submit in the interest of time.

I think Steven tested that already in lines 100-112.

**Pull request merged**